savant/deepstream/meta/object.py

```diff
@@ -241,6 +242,35 @@ def label(self, value: str):
                 f'the value is `{type(value)}`'
             )
 
+    @property
+    def draw_label(self) -> str:
+        custom_data_struct = nvds_get_obj_custom_data_struct(self.ds_object_meta)
+        if custom_data_struct:
+            return pyds.get_string(custom_data_struct.message)
+        return self.label
+
+    @draw_label.setter
+    def draw_label(self, value: str):
+        if isinstance(value, str):
+            data = nvds_get_obj_custom_data_struct(self.ds_object_meta)
+            if data:
+                data.message = value
+            else:
+                user_meta = pyds.nvds_acquire_user_meta_from_pool(
+                    self._frame_meta.base_meta.batch_meta
+                )
+                if user_meta:
+                    data = pyds.alloc_custom_struct(user_meta)
+                    data.message = value
+                    user_meta.user_meta_data = data
+                    user_meta.base_meta.meta_type = pyds.NvDsMetaType.NVDS_USER_META
```

Comment:
It is very dangerous to use the common meta type NVDS_USER_META. Any addition of data of this type will lead to errors or hard to track problems. Use nvds_get_user_meta_type https://docs.nvidia.com/metropolis/deepstream/python-api/PYTHON_API/Methods/methodsdoc.html#pyds.nvds_get_user_meta_type to generate your own unique user meta type.

Reply:
Added custom user meta type.